kuma/static/js/components/payments/focus-visible.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    'use strict';
+    var contributeForm = document.getElementById('contribute-form');
+
+    function removeFocusVisible() {
+        this.classList.remove('focus-visible');
+        this.removeEventListener('focusout', removeFocusVisible);
+    }
+
+    // When a keyup event is triggered on the contribute form,
+    contributeForm.addEventListener('keyup', function(event) {
```

Comment:
IIUC, when the user clicks on an input field, it will not display a focus ring. But as soon as they start typing into the field, the focus ring will appear. What if you only added the focus-visible class if the key up event was for the Tab key. Would that work?

Reply:
Yup, that would definitely be an improvement. I will update the code.